core/orch_test.go

```diff
@@ -502,10 +503,12 @@ func TestProcessPayment_GivenRecipientError_ReturnsNil(t *testing.T) {
 
 func TestProcessPayment_GivenNoSender_ReturnsError(t *testing.T) {
 	n, _ := NewLivepeerNode(nil, "", nil)
+	n.Balances = NewBalances(5 * time.Second)
 	recipient := new(pm.MockRecipient)
 	n.Recipient = recipient
 	orch := NewOrchestrator(n)
 	recipient.On("ReceiveTicket", mock.Anything, mock.Anything, mock.Anything).Return("some sessionID", false, nil)
+	recipient.On("EV").Return(big.NewRat(1000000000, 1))
```

Comment:
Do we need this mock? `ProcessPayment()` should return before getting to the `EV()` call

Reply:
Correct, `n.Balances = NewBalances(5 * time.Second)` as well.

Reply:
fixed in https://github.com/livepeer/go-livepeer/pull/956/commits/794632acb80531d8222ca2a12334d3ddcd321427

Reply:
Looks like the `n.Balances` initialization on L506 still needs to be removed?